deployer/conf/elasticsearch-logging.yml

```diff
@@ -1,16 +1,25 @@
 # you can override this using by setting a system property, for example -Des.logger.level=DEBUG
 es.logger.level: INFO
-rootLogger: ${es.logger.level}, console
+rootLogger: ${es.logger.level}, file
 logger:
   # log action execution errors for easier debugging
   action: WARN
 
+  # deprecation logging, turn to DEBUG to see them
+  deprecation: WARN, deprecation_log_file
```

Comment:
It means that you will only see deprecation log events in the file that are of a log level `WARN` or higher.  So most likely nothing since we don't mix versions of clients within the cluster.

FWIW, I run using `INFO` on my v2.4.x staging instance and I have seen nothing at all.